src/clr-addons/components.clr-addons.scss

```diff
@@ -78,6 +78,27 @@ body,
     @include card-active-after();
   }
 
+  // firefox needs this, because the hover in @media (hover: hover) is not supported
+  @-moz-document url-prefix() {
```

Comment:
Please put the content of both queries in a mixin and use it in both to prevent multiple maintenance